src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -61,7 +64,9 @@ public void navigate(final Project project) {
       final FlutterSdk sdk = FlutterSdk.getFlutterSdk(project);
       if (sdk != null) {
         try {
-          sdk.run(FlutterSdk.Command.DOCTOR, module, project.getBaseDir(), null);
+          final VirtualFile contentRoot = ModuleUtil.getProjectContentRoot(module);
+          assert contentRoot != null;
+          sdk.run(FlutterSdk.Command.DOCTOR, module, contentRoot, null);
```

Comment:
In this case `flutter doctor` can actually take advantage of directory context so it would be most ideal to provide it.   Agreed though that we can remove `ModuleUtil`.  Stay tuned...